### PR TITLE
FEAT: redux toolkit setup

### LINKED ISSUE #16: redux toolkit setup
### Requirement

- Do the initial basic setup for redux toolkit
- follow this guide: https://redux-toolkit.js.org/introduction/getting-started

### PR DESCRIPTION
**Pull request by samyakshah3008**
Basic redux-toolkit setup done ✅ 
Resolves #16

### CODE REVIEW

**Review by vvaibhavdesai — APPROVED** [review]
Looks good great work 🚀

File: package.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@reduxjs/toolkit": "^1.9.1",
```
Comment (vvaibhavdesai) [inline package.json:13]: just one question why 1.9.1 and not 1.9.2 ?

Comment (samyakshah3008) [reply package.json]: New version just cam out today, upgraded to latest.

**Review by vinayak-trivedi — APPROVED** [review]
LGTM

**Review by Maheima — APPROVED** [review]
LGTM!

**Pull request merged**